src/specific/file.c

```diff
@@ -273,12 +286,50 @@ static int32_t LoadObjects(MYFILE *fp)
     AnimRanges = game_malloc(
         sizeof(ANIM_RANGE_STRUCT) * AnimRangeCount, GBUF_ANIM_RANGES);
     FileRead(AnimRanges, sizeof(ANIM_RANGE_STRUCT), AnimRangeCount, fp);
+    for (int32_t i = 0; i < AnimRangeCount; ++i) // DAN
```

Comment:
What does DAN mean?…

Reply:
that I changed it so I can find it again, removed.